.travis.yml

```diff
@@ -4,6 +4,7 @@ python:
   - "2.7_with_system_site_packages"
   - "3.2_with_system_site_packages"
   - "3.3"
+  - "3.4"
```

Comment:
Probably it hasn't been tested back then. I just 'merged' the previous efforts.
And I haven't tested with 3.5 myself, so can't rightly add it here.